pkg/config/config.go

```diff
@@ -27,6 +27,7 @@ import (
 )
 
 const DefaultConfigurationFileName = "v3io.yaml"
+const SCHEMA_CONFIG = "/.schema"
```

Comment:
`/` should be removed
Use `filePath.Join(...)` instead (see https://golang.org/src/path/filepath/path.go?s=5895:5927#L199)

Reply:
As noted in another comment, the correct function we ought to be using is actually `path.Join`.